### PR TITLE
Fix indentation in YAML examples

### PR DESCRIPTION
**Pull request by afrittoli**
Fix indentation in task YAML examples the developers README.

### CODE REVIEW

**Comment by mgreau** [comment]
For the YAML examples, I noticed this behavior a couple of times while using the examples. It seems related to the @mattmoor-sockpuppet bot as you can see here https://github.com/knative/build-pipeline/commit/b4d1882702f67731c399f658ce75f647444b3ac5#diff-40d8c60440f598fd4b4bb12eb4e35e24

IMO this is something to fix on the bot side before merging this PR. cc @mattmoor

**Comment by mgreau** [comment]
> It seems related to the @mattmoor-sockpuppet bot 

To be clear, the [bot config is here](https://github.com/mattmoor/knobots/blob/master/config/prettier-template.yaml#L11-L15), so it seems be an issue with [prettier.](https://github.com/prettier/prettier/issues)

**Comment by afrittoli** [comment]
Good catch @mgreau, thanks. I 100% agree.

File: docs/developers/README.md [hunk 1]
```diff
@@ -88,10 +88,10 @@ expected in directory path `/workspace/output/resource_name`.
   inputs:
     resources:
       - name: gcs-workspace
-    type: storage
-    name: storage
+        type: storage
+        name: storage
```
Comment (shashwathi) [inline docs/developers/README.md]: `name` key is repeated here. I notice that its already in old yaml file but lets take this opportunity to fix this

**Comment by mgreau** [comment]
@afrittoli ah ok good to know, thanks.

**Comment by bobcatfish** [comment]
Whoa nice, thanks for fixing this @afrittoli - and for the vigilance @mgreau following up about @mattmoor-sockpuppet !! :heart: 

/lgtm
/approve
/meow space

**Pull request merged**